releases.md

```diff
@@ -55,29 +55,36 @@ Further documentation available:
 - **Estimated End of Life**: 2024-01-24
 - **Patch Releases**: [v0.44.0][v0-44-0]
 
-### v0.43
-
-- **Latest Release**: [v0.43.2][v0-43-2] (2023-01-10) ([docs][v0-43-2-docs], [examples][v0-43-2-examples])
-- **Initial Release**: [v0.43.0][v0-43-0] (2022-12-22)
-- **Estimated End of Life**: 2023-04-22
-- **Patch Releases**: [v0.43.0][v0-43-0], [v0.43.1][v0-43-1], [v0.43.2][v0-43-2]
- 
 ### v0.41 (LTS)
 
 - **Latest Release**: [0.41.2][v0-41-2] (2023-04-06) ([docs][v0-41-2-docs], [examples][v0-41-2-examples])
 - **Initial Release**: [v0.41.0][v0-41-0] (2022-10-31)
-- **Estimated End of Life**: 2023-10-30
+- **End of Life**: 2023-10-30
 - **Patch Releases**: [v0.41.0][v0-41-0], [v0.41.1][v0-41-1], [v0.41.2][v0-41-2]
 
 ## End of Life Releases
 
+### v0.46
+
+- **Latest Release**: [v0.46.0][v0-46-0] (2023-03-17) ([docs][v0-46-0-docs], [examples][v0-46-0-examples])
+- **Initial Release**: [v0.46.0][v0-46-0] (2023-03-17)
+- **Estimated End of Life**: 2023-04-17
```

Comment:
```suggestion
- **End of Life**: 2023-04-24
```

Reply:
Thanks, good idea, we shouldn't say estimated once they are EOL 😅